torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1249,6 +1249,12 @@ def method_tests():
         ('qr', (3, 2, S, S), (False,), 'square_many_batched', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
         ('qr', (3, 2, S, S - 2), (True,), 'tall_many_batched', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
         ('qr', (3, 2, S - 2, S), (True,), 'wide_many_batched', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
+        ('lu', (S, S), (True, False), 'square_single_no_info', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
+        ('lu', (S, S), (True, True), 'square_single_with_info', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
+        ('lu', (3, S, S), (True, False), 'square_batch_no_info', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
+        ('lu', (3, S, S), (True, True), 'square_batch_with_info', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
+        ('lu', (3, 3, S, S), (True, False), 'square_many_batches_no_info', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
+        ('lu', (3, 3, S, S), (True, True), 'square_many_batches_with_info', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
```

Comment:
Again jit causes issues. @albanD , could you please advice something here?

Reply:
Is it the same "cannot script custom Function" issue as for lobpcg?
If so I think we can use the same strategy of not allowed scripting when it is differentiated.

Reply:
Looks like. I will adopt it, but then I will not be able to put tests into `common_methods_invocations,py`.

Reply:
Ho, isn't one of the random boolean in this big tuple disabling jit tests for that entry? :D

Reply:
Apparently, not in the `common_methods_invocations.py`. I checked `test_jit.py` and it does import something like EXCLUDE_GRADCHECK from `common_methods_invokations.py`.